filers/default/src/main/java/org/commonjava/indy/filer/def/conf/DefaultStorageProviderConfiguration.java

```diff
@@ -105,4 +125,59 @@ public Properties getSystemPropertyAdditions()
         p.setProperty( NFS_STORAGE_DIR, getStorageRootDirectory().getAbsolutePath() );
         return p;
     }
+
+
+    // Path mapped storage config
+    private String cassandraHost = "localhost";
+
+    @ConfigName( "storage.cassandra.host" )
+    public void setCassandraHost( String host )
+    {
+        cassandraHost = host;
+        cassandraProps.put( PROP_CASSANDRA_HOST, cassandraHost );
+    }
+
+    private int cassandraPort = 9042;
+
+    @ConfigName( "storage.cassandra.port" )
+    public void setCassandraPort( int port )
+    {
+        cassandraPort = port;
+        cassandraProps.put( PROP_CASSANDRA_PORT, cassandraPort );
+    }
+
+    private String cassandraKeyspace = "indy";
+
+    @ConfigName( "storage.cassandra.keyspace" )
+    public void setCassandraKeyspace( String keyspace )
+    {
+        cassandraKeyspace = keyspace;
+        cassandraProps.put( PROP_CASSANDRA_KEYSPACE, cassandraKeyspace );
+    }
+
+    private PathMappedStorageConfig pathMappedStorageConfig;
+
+    @Override
+    public int getGCIntervalInMinutes()
```

Comment:
Are the corresponding setter methods exposed somewhere for tuning from Indy config files?

Reply:
Those are missing here. Will add them.